flink-connector-elasticsearch8/src/main/java/org/apache/flink/connector/elasticsearch/sink/Elasticsearch8AsyncSinkBuilder.java

```diff
@@ -100,16 +111,62 @@ public Elasticsearch8AsyncSinkBuilder<InputT> setHeaders(Header... headers) {
     }
 
     /**
-     * setCertificateFingerprint set the certificate fingerprint to be used to verify the HTTPS
-     * connection.
+     * Allows to bypass the certificates chain validation and connect to insecure network endpoints
+     * (for example, servers which use self-signed certificates).
+     *
+     * @return this builder
+     */
+    public Elasticsearch8AsyncSinkBuilder<InputT> allowInsecure() {
```

Comment:
:+1: Looks like a straightforward backport of ES6/7 impl, thank you

Reply:
Thanks, I refactored this ES 8 a bit in #100 to assist this.